doc/jvm/xgboost_spark_migration.rst

```diff
@@ -63,73 +63,26 @@ Learning to rank using XGBoostRegressor has been replaced by a dedicated `XGBoos
 to support ranking algorithms.
 
 .. code-block:: scala
-  
-  // before 3.0
+
+  // before xgboost4j-spark 3.0
   val regressor = new XGBoostRegressor().setObjective("rank:ndcg")
 
-  // after 3.0
+  // after xgboost4j-spark 3.0
   val ranker = new XGBoostRanker()
 
-******************************
```

Comment:
Is this no longer needed?

Reply:
yes, I made https://github.com/dmlc/xgboost/pull/10845 to be compatible with old style.